src/tests/compiler/statement/statementedNodeTests.ts

```diff
@@ -214,6 +214,13 @@ describe(nameof(StatementedNode), () => {
             doFirstChildTest<DefaultClause>(defaultClause, 3, "newText;\nsecondText;", 2,
                 "switch (x) {\n    default:\n        x = 0;\n        y = 1;\n        break;\n        newText;\n        secondText;\n}", SyntaxKind.DefaultClause);
         });
+
+        it("should insert statements in a Block", () => {
+            const {sourceFile, firstChild} = getInfoFromTextWithSyntax<Block>  ('function():number{const a = 1, b = true;}', SyntaxKind.Block);
```

Comment:
Could you run `npm run lint` and then fix the linting errors?

Reply:
npm run lint didn't reported any errors. Did you expected one ?

Reply:
Really? That's strange. Yeah it should for the single quotes and spacing. Should be double quotes and no spaces between `<Block>` and the open paren.